### PR TITLE
V0.2

### PR DESCRIPTION
**Pull request by JarlDysnomia**
### ADDED
FIA Water Supply
FIA Mortar Pit

NATO Refuel & Rearm
AAF Refuel & Rearm
CSAT Refuel & Rearm

CSAT Artillery Position
AAF QRF
AAF Checkpoint
Vehicle & Personnel Decontamination

IDAP Field Hospital
IED Factory
Transmitter

### CODE REVIEW

File: Source/JCC/config.cpp [hunk 1]
```diff
@@ -146,6 +146,35 @@ class CfgPatches
 					class Object52 {side=8;vehicle="Land_SandbagBarricade_01_half_F";rank="";position[]={-11.9551,-8.33569,-0.00143909};dir=179.401;};
 					class Object53 {side=8;vehicle="Land_SandbagBarricade_01_F";rank="";position[]={-12.5815,8.14771,-0.00143909};dir=270.586;};
 				};
+				class NATO_Refuel_&_Rearm
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Don't use special characters in class-names. Should be `natoRefuelRearm`

File: Source/JCC/config.cpp [hunk 2]
```diff
@@ -205,8 +234,114 @@ class CfgPatches
 					class Object21 {side=8;vehicle="Land_SurvivalRadio_F";rank="";position[]={-5.19482,-3.10571,-0.00143909};dir=73.1444;};
 					class Object22 {side=8;vehicle="Land_Garbage_square5_F";rank="";position[]={-7.80664,-1.8374,-0.00143909};dir=181.548;};
 				};
-			};
-			class IDAP 
+				class FIA_Water_Supply
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Same thing here, **no underscores**.

File: Source/JCC/config.cpp [hunk 3]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object37 {side=8;vehicle="Land_MoneyBills_01_bunch_F";rank="";position[]={-3.70947,-4.91064,0.121269};dir=0;};
+					class Object38 {side=8;vehicle="Land_BakedBeans_F";rank="";position[]={-4.51709,-4.19922,0.121269};dir=0;};
+					class Object39 {side=8;vehicle="Land_RiceBox_F";rank="";position[]={-4.4707,-4.38721,0.121269};dir=0;};
+					class Object40 {side=8;vehicle="Land_RiceBox_F";rank="";position[]={-4.54883,-4.49219,0.121269};dir=0;};
+					class Object41 {side=8;vehicle="Land_PaperBox_closed_F";rank="";position[]={4.04297,-5.04639,-0.00143909};dir=0;};
+					class Object42 {side=8;vehicle="Land_TinContainer_F";rank="";position[]={-4.45996,-4.68457,0.121269};dir=231.624;};
+					class Object43 {side=8;vehicle="Land_Garbage_square3_F";rank="";position[]={5.08838,-4.229,-0.00143909};dir=349.631;};
+					class Object44 {side=8;vehicle="ModuleCurator_F";rank="";position[]={41.6118,-12.9966,-0.00143909};dir=0;};
+				};
+				class Transmitter
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: This is too generically named.

File: Source/JCC/config.cpp [hunk 4]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object18 {side=8;vehicle="Land_Net_Fence_8m_F";rank="";position[]={-0.410645,8.96582,-0.00143909};dir=0;};
+					class Object19 {side=8;vehicle="Land_PowLines_Transformer_F";rank="";position[]={-7.15479,6.07129,-0.00143909};dir=0;};
+					class Object20 {side=8;vehicle="Land_PowLines_Transformer_F";rank="";position[]={-7.04004,8.18506,-0.00143909};dir=0;};
+					class Object21 {side=8;vehicle="Land_TTowerBig_2_F";rank="";position[]={4.4126,5.26807,-0.00143909};dir=270.213;};
+				};
+			};
+			class CSAT
+			{
+				name = "CSAT";
+				class CSAT_Artillery
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Underscores.

File: Source/JCC/config.cpp [hunk 5]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object8 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={-4.05762,4.52441,-0.00143909};dir=89.7253;};
+					class Object9 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={5.24414,4.33643,-0.00143909};dir=89.7253;};
+					class Object10 {side=8;vehicle="O_MBT_02_arty_F";rank="";position[]={0.614746,6.75293,-0.0890951};dir=0.000213077;};
+					class Object11 {side=8;vehicle="O_Truck_02_fuel_F";rank="";position[]={-6.81982,-2.7417,-0.0061903};dir=353.115;};
+					class Object12 {side=8;vehicle="Land_HBarrier_3_F";rank="";position[]={-2.51074,11.0479,-0.00143909};dir=113.937;};
+					class Object13 {side=8;vehicle="Land_HBarrier_3_F";rank="";position[]={3.35156,10.8926,-0.00143909};dir=58.9842;};
+					class Object14 {side=8;vehicle="Land_HBarrier_5_F";rank="";position[]={-1.70166,10.8901,-0.00143909};dir=0;};
+					class Object15 {side=8;vehicle="ModuleCurator_F";rank="";position[]={-15.6704,3.83984,-0.00143909};dir=0;};
+				};
+				class CSAT_Refuel_&_Rearm
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: See my review on the NATO classname.

File: Source/JCC/config.cpp [hunk 6]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object21 {side=8;vehicle="PortableHelipadLight_01_white_F";rank="";position[]={7.63574,-9.85645,-0.00143909};dir=359.996;};
+					class Object22 {side=8;vehicle="PortableHelipadLight_01_white_F";rank="";position[]={6.02148,-11.0884,-0.00143909};dir=0.00185138;};
+					class Object23 {side=8;vehicle="PortableHelipadLight_01_white_F";rank="";position[]={7.59277,-11.105,-0.00143909};dir=0.00177036;};
+					class Object24 {side=8;vehicle="PortableHelipadLight_01_yellow_F";rank="";position[]={4.61816,-14.0474,-0.00143909};dir=359.996;};
+					class Object25 {side=8;vehicle="PortableHelipadLight_01_yellow_F";rank="";position[]={10.8325,-10.4863,-0.00143909};dir=359.996;};
+			};
+			class AAF
+			{
+				name = "AAF";
+				class AAF_QRF
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Underscore.

File: Source/JCC/config.cpp [hunk 7]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object50 {side=8;vehicle="Land_Cargo_House_V1_F";rank="";position[]={18.7837,4.30273,-0.00143909};dir=26.3289;};
+					class Object51 {side=8;vehicle="Land_HelipadCircle_F";rank="";position[]={15.9844,-10.5342,-0.00143909};dir=0;};
+					class Object52 {side=8;vehicle="Land_Sleeping_bag_brown_F";rank="";position[]={19.0415,7.00195,-0.00128698};dir=88.649;};
+					class Object53 {side=8;vehicle="I_Heli_light_03_dynamicLoadout_F";rank="";position[]={16.0186,-12.3643,-0.00667381};dir=0.00076681;};
+					class Object54 {side=8;vehicle="Land_DeskChair_01_olive_F";rank="";position[]={20.3882,2.62305,-0.00244093};dir=317.575;};
+					class Object55 {side=8;vehicle="Land_Pillow_grey_F";rank="";position[]={19.3105,7.19824,-0.028028};dir=226.869;};
+					class Object56 {side=8;vehicle="Land_DeskChair_01_olive_F";rank="";position[]={20.8062,3.73975,-0.00244141};dir=255.991;};
+					class Object57 {side=8;vehicle="PortableHelipadLight_01_blue_F";rank="";position[]={19.6069,-12.6421,-0.00143909};dir=359.996;};
+				};
+				class AAF_Checkpoint
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Underscore.

File: Source/JCC/config.cpp [hunk 8]
```diff
@@ -434,6 +628,476 @@ class CfgPatches
...
+					class Object143 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={-24.209,11.793,-0.00143909};dir=91.1435;};
+					class Object144 {side=8;vehicle="Land_SolarPanel_2_F";rank="";position[]={20.1563,17.3066,-0.00143909};dir=0;};
+					class Object145 {side=8;vehicle="Flag_AAF_F";rank="";position[]={19.7515,-19.9785,-0.00143909};dir=0;};
+					class Object146 {side=8;vehicle="Flag_AAF_F";rank="";position[]={-21.0483,-20.2271,-0.00143909};dir=0;};
+					class Object147 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={22.6499,-18.3564,-0.00143909};dir=91.1435;};
+					class Object148 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={19.5063,-21.9624,-0.00143909};dir=0;};
+					class Object149 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={-20.6504,-22.0488,-0.00143909};dir=0;};
+					class Object150 {side=8;vehicle="Land_HBarrier_Big_F";rank="";position[]={-23.8833,-18.8198,-0.00143909};dir=91.1435;};
+				};
+				class AAF_Refuel_&_Rearm
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: See my previous comments.

File: Source/JCC/config.cpp [hunk 9]
```diff
@@ -442,6 +1106,30 @@ class CfgPatches
 		
 	
 
+
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: Too many new-lines. Remove all these. There should be only **one** new-line at the end of your code.

**Review by Radiumio — CHANGES_REQUESTED** [review]
Please see all my comments and make changes as necessary.

File: Source/JCC/config.cpp [hunk 10]
```diff
@@ -581,7 +581,7 @@ class CfgPatches
 					class Object12 {side=8;vehicle="RoadBarrier_F";rank="";position[]={20.2012,-2.88428,-0.00544024};dir=91.0658;};
 					class Object13 {side=8;vehicle="RoadCone_L_F";rank="";position[]={20.126,-5.87598,-0.00199223};dir=267.326;};
 				};
-				class Razorwire_Barricade
+				class RazorwireBarricade
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: **r**azorwireBarricade

**Review by Radiumio — CHANGES_REQUESTED** [review]

File: Source/JCC/config.cpp [hunk 11]
```diff
@@ -205,8 +234,114 @@ class CfgPatches
...
+					class Object46 {side=8;vehicle="Land_GarbageHeap_01_F";rank="";position[]={-5.62109,11.0381,-0.00144053};dir=355.342;};
+					class Object47 {side=8;vehicle="Land_PortableSpeakers_01_F";rank="";position[]={11.2729,4.854,-0.00354528};dir=320.755;};
+					class Object48 {side=8;vehicle="Land_Laptop_03_sand_F";rank="";position[]={11.3408,5.21289,-0.00340748};dir=76.5516;};
+					class Object49 {side=8;vehicle="Land_WoodenTable_large_F";rank="";position[]={11.6772,4.7041,-0.00243998};dir=0.00123817;};
+					class Object50 {side=8;vehicle="Land_GarbageBags_F";rank="";position[]={10.939,-6.41113,-0.00143909};dir=0;};
+					class Object51 {side=8;vehicle="Flag_FIA_F";rank="";position[]={10.3716,6.67676,-0.00143909};dir=0;};
+					class Object52 {side=8;vehicle="WaterPump_01_forest_F";rank="";position[]={13.7271,4.57031,-0.00058794};dir=360;};
+					class Object53 {side=8;vehicle="Land_Slum_05_F";rank="";position[]={7.21143,15.979,-0.00143909};dir=0;};
+					class Object54 {side=8;vehicle="B_G_Van_02_vehicle_F";rank="";position[]={14.1147,14.5654,0.0839167};dir=19.096;};
+			    };
```
Comment (Radiumio) [inline Source/JCC/config.cpp:294]: Align this bracket properly.

File: Source/JCC/config.cpp [hunk 12]
```diff
@@ -434,34 +628,477 @@ class CfgPatches
...
+					class Object37 {side=8;vehicle="Land_MoneyBills_01_bunch_F";rank="";position[]={-3.70947,-4.91064,0.121269};dir=0;};
+					class Object38 {side=8;vehicle="Land_BakedBeans_F";rank="";position[]={-4.51709,-4.19922,0.121269};dir=0;};
+					class Object39 {side=8;vehicle="Land_RiceBox_F";rank="";position[]={-4.4707,-4.38721,0.121269};dir=0;};
+					class Object40 {side=8;vehicle="Land_RiceBox_F";rank="";position[]={-4.54883,-4.49219,0.121269};dir=0;};
+					class Object41 {side=8;vehicle="Land_PaperBox_closed_F";rank="";position[]={4.04297,-5.04639,-0.00143909};dir=0;};
+					class Object42 {side=8;vehicle="Land_TinContainer_F";rank="";position[]={-4.45996,-4.68457,0.121269};dir=231.624;};
+					class Object43 {side=8;vehicle="Land_Garbage_square3_F";rank="";position[]={5.08838,-4.229,-0.00143909};dir=349.631;};
+					class Object44 {side=8;vehicle="ModuleCurator_F";rank="";position[]={41.6118,-12.9966,-0.00143909};dir=0;};
+				};
+				class Transmitter1
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: **`t`**ransmitter1

File: Source/JCC/config.cpp (hunk 11 above)
Comment (JarlDysnomia) [reply Source/JCC/config.cpp:294]: It is fine on my side.

**Comment by Radiumio** [comment]
I fixed several issues you overlooked. Fetch the new changes and you can try test it before we merge.

**Review by Radiumio — APPROVED** [review]

**Pull request merged**